tests/functional/bin/run_sql_task_test.py

```diff
@@ -10,8 +10,9 @@
 
 
 class TestRunSQLTask:
-    def test_sanity(self, environ):
-        self.run_task(environ, "hello_world")
+    @pytest.mark.parametrize("task_name", ("hello_world", "report/create_dev_users"))
+    def test_it(self, environ, task_name):
+        self.run_task(environ, task_name)
```

Comment:
Unlike the other tasks, the `report/create_dev_users` task should run on it's own succesfully.